ydb/core/tx/schemeshard/schemeshard__operation_create_external_data_source.cpp

```diff
@@ -118,13 +35,13 @@ class TPropose: public TSubOperationState {
         LOG_I(DebugHint() << "HandleReply TEvOperationPlan"
             << ": step# " << step);
 
-        TTxState* txState = context.SS->FindTx(OperationId);
+        const TTxState* txState = context.SS->FindTx(OperationId);
         Y_ABORT_UNLESS(txState);
         Y_ABORT_UNLESS(txState->TxType == TTxState::TxCreateExternalDataSource);
 
-        auto pathId = txState->TargetPathId;
-        auto path = TPath::Init(pathId, context.SS);
-        TPathElement::TPtr pathPtr = context.SS->PathsById.at(pathId);
+        const auto pathId  = txState->TargetPathId;
+        const auto path    = TPath::Init(pathId, context.SS);
+        const auto pathPtr = context.SS->PathsById.at(pathId);
```

Comment:
Why auto is better than explicit `TPathElement::TPtr`?

Reply:
Reverted change